ledger-service/jwt/src/main/scala/com/digitalasset/jwt/Base64.scala

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.jwt
+
+import scalaz.{Show, \/}
+
+object Base64 {
```

Comment:
If this compiles, as it seems to be a local utility:

```suggestion
private object Base64 {
```

Reply:
I compiles.